tests/php/src/MobileRedirectionTest.php

```diff
@@ -476,13 +476,33 @@ public function test_set_mobile_redirection_disabled_cookie() {
 		$this->assertArrayNotHasKey( MobileRedirection::DISABLED_STORAGE_KEY, $_COOKIE );
 	}
 
-	/** @covers ::add_mobile_redirect_script() */
+	/**
+	 * @covers ::add_mobile_redirect_script()
+	 * @covers ::get_inline_script_tag()
+	 * @covers ::sanitize_script_attributes()
+	 */
 	public function test_add_mobile_redirect_script() {
 		ob_start();
 		$this->instance->add_mobile_redirect_script();
 		$output = ob_get_clean();
+		$this->assertRegExp( '#<script\b[^>]*>#', $output );
```

Comment:
Could this test be made more specific? We wouldn't want any unwanted attributes to slip through.

```suggestion
		$this->assertRegExp( '#<script type="text/javascript">#', $output );
```

Reply:
If the theme supports HTML5 then this attribute will not be present. But I guess we aren't testing with a theme active, so I guess it can't hurt.